test-it/karma/javascriptinvoke/InternalFunction.Test.js

```diff
@@ -62,6 +62,7 @@ describe('A JavaScript function invoke', function () {
             expect(viPathParser('error2.status')).toBeTrue();
             expect(viPathParser('error2.code')).toBe(777);
             expect(viPathParser('error2.source')).toContain('this is the error message');
+            expect(viPathParser('error2.source')).toContain('in MyVI');
```

Comment:
I think we need to check for "NI_InternalFunctionSetsError" to be contained on the error string. That is the purpose of the change you're making.